core/sched.c

```diff
@@ -156,6 +156,10 @@ thread_t *__attribute__((used)) sched_run(void)
     thread_t *next_thread = container_of(sched_runqueues[nextrq].next->next,
                                          thread_t, rq_entry);
 
+#if (IS_USED(MODULE_SCHED_RUNQ_CALLBACK))
```

Comment:
Would you mind to sneak in the style fix suggested by the CI for lines 78/79? (But as separate commit, to ease potential reverts.)